### PR TITLE
Feature/section video

### PR DESCRIPTION
**Pull request by thanhtai2707**
- [x] Background
- [x] Title
- [x] Button
image

### CODE REVIEW

File: practice-01/index.html [hunk 1]
```diff
@@ -62,6 +61,18 @@ <h1 class="title">
         <div class="diamond diamond-primary"></div>
         <div class="diamond diamond-secondary"></div>
       </div>
+      <div class="container-marketing">
```
Comment (phattruong-agilityio) [inline practice-01/index.html:64]: what is `container-marketing` ?

Comment (phattruong-agilityio) [reply practice-01/index.html:64]: This one should be section

File: practice-01/index.html [hunk 2]
```diff
@@ -62,6 +61,18 @@ <h1 class="title">
...
+        <section class="video">
+          <div class="video-img">
+            <div class="title title-banner">
+              <h1 class="title title-banne title-video">
```
Comment (phattruong-agilityio) [inline practice-01/index.html:68]: Too many h1 in a page

File: practice-01/assets/styles/index.css [hunk 3]
```diff
@@ -127,3 +127,42 @@
   left: 193px;
 }
 
+.container-marketing {
```
Comment (ThanhPhat1080) [inline practice-01/assets/styles/index.css:130]: `.container-marketing` ??

File: practice-01/index.html [hunk 4]
```diff
@@ -62,6 +61,18 @@ <h1 class="title">
         <div class="diamond diamond-primary"></div>
         <div class="diamond diamond-secondary"></div>
       </div>
+      <section class="marketing-section">
+        <section class="video">
```
Comment (ThanhPhat1080) [inline practice-01/index.html:65]: @thanhtai2707 I told you it's not a `video`

File: practice-01/index.html [hunk 5]
```diff
@@ -62,6 +61,18 @@ <h1 class="title">
...
+          <div class="video-img">
+            <div class="title title-banner">
+              <h2 class="title title-banne title-video">
```
Comment (ThanhPhat1080) [inline practice-01/index.html:68]:
```html
<div class="title title-banner">
              <h2 class="title title-banne title-video">
```
Why have 2 "title title-banner" here. is `h2` need it

File: practice-01/index.html [hunk 6]
```diff
@@ -62,6 +61,18 @@ <h1 class="title">
...
+                reach more customers
+              </h2>
+              <button class="btn btn-secondary btn-secondary-bottom">learn how</button>
```
Comment (ThanhPhat1080) [inline practice-01/index.html:71]: `btn btn-secondary btn-secondary-bottom`
 why have `btn-secondary-bottom` here. ??

File: practice-01/index.html (hunk 4 above)
Comment (ThanhPhat1080) [reply practice-01/index.html:65]: btw it's has 2  sections. I don't know why

**Pull request merged**